functions/send_welcome_message.ts

```diff
@@ -53,6 +49,4 @@ const setupFunction: SlackFunctionHandler<
   return await {
```

Comment:
```suggestion
  return {
```
Unless there's something I'm not understanding about the code as written, `await` shouldn't be necessary? 🤔